### DISCUSSION TITLE
Where is contributor discussion happening ?

### DISCUSSION THREAD

**Discussion by johnoscott** [Milkdown/milkdown#80] [General]
Hi, I am keen to contribute PRs and discussion of features to this project. Where is day-to-day discussion happening ? Here (Github discussions) or do you have a Discord or Slack. If you haven't set either up yet can I suggest creating a Slack account.

Right now I am trying to keep up with the frequent commits and versions being generated. The comments are short and I dont really understand what has been done (e.g. 804cbb5: Add support for clipboard and collaborative abilities. - sounds exciting but I have no idea what it means).

**Comment by Saul-Mirone** [comment]
Make a lot sense, we surely need a discuss channel, let's use discord since I see the typescript discord group works very well(and discord is really cool😃)

Here is link

https://discord.gg/BgepcQNu

**Comment by Saul-Mirone** [comment]
BTW, I shared a lot of key features in this Twitter list:

https://twitter.com/i/events/1417705730086342666?s=21
